### DISCUSSION TITLE
Colours, italics and bold for bookmarks -- IMPLEMENTED

### DISCUSSION THREAD

**Discussion by chapmanwilliam** [pymupdf/PyMuPDF#799] [Ideas]
PyMuPDF doesn't reveal this information with doc.getToC(). 

Nor is it possible to write it back to the pdf.

This would be a very useful feature for me. And presumably a fairly simple feature request...?

William

**Comment by JorjMcKie** [comment]
> And presumably a fairly simple feature request...?

Not really. Or let's say: not difficult, but quite a lot of work!
I will put it on my list of enhancements.
